src/main/java/jsr223/docker/compose/utils/DockerComposePropertyLoader.java

```diff
@@ -83,20 +79,18 @@ private DockerComposePropertyLoader()  {
         this.dockerHost = properties.getProperty("docker.host", "");
     }
 
+    public static DockerComposePropertyLoader getInstance() {
+        return DockerComposePropertyLoaderHolder.INSTANCE;
+    }
 
-
-
-     /**    Initializes DockerComposePropertyLoader.
-
-      DockerComposePropertyLoaderHolder is loaded on the first execution of DockerComposePropertyLoader.getInstance()
-      or the first access to DockerComposePropertyLoaderHolder.INSTANCE, not before.
-    **/
-     @NoArgsConstructor(access = AccessLevel.PRIVATE)
+    /**
+     * Initializes DockerComposePropertyLoader.
+     * <p>
+     * DockerComposePropertyLoaderHolder is loaded on the first execution of DockerComposePropertyLoader.getInstance()
+     * or the first access to DockerComposePropertyLoaderHolder.INSTANCE, not before.
+     **/
+    @NoArgsConstructor(access = AccessLevel.PRIVATE)
     private static class DockerComposePropertyLoaderHolder {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1118)